### ISSUE TITLE
SoftMax

### ISSUE THREAD

**Issue by takenmore** [wlkdb/TensorflowInAndroid#1]
一点点瑕疵  模型好像是没使用softmax转概率。。
